src/libraries/pkg/baseline/packageIndex.json

```diff
@@ -28,7 +35,7 @@
       "InboxOn": {},
       "AssemblyVersionInPackageVersion": {
         "1.0.0.0": "1.0.0",
-        "5.0.0.0": "6.0.0"
+        "6.0.0.0": "6.0.0"
```

Comment:
This and many below are missing entries for "5.0.0.0": "5.0.0".   I think everything that existed during 5.0 timeframe should get both entries.

Reply:
And looks like most of these are also missing 5.0.0 to be listed as Stable Version. Is that going to happen after we ship or should that be includded on this PR as well?

Reply:
That should be included here.

Reply:
Yes i am planning to include that after we release tomorrow